aptos/contracts/sources/governance/set_update_fee.move

```diff
@@ -0,0 +1,43 @@
+module pyth::set_update_fee {
+    use wormhole::cursor;
+    use pyth::deserialize;
+    use std::math64;
+    use pyth::state;
+
+    const MAX_U64: u128 = (1 << 64) - 1;
+
+    const E_OVERFLOW: u64 = 15;
+
+    struct SetUpdateFee {
```

Comment:
the ship may have sailed on this already... but it sure seems like it would be simpler to pass a u64 directly instead of using a floating point representation

Reply:
The reasoning behind this was so that we could use the same shaped payload for all the runtimes, while being able to:

- Deserialize integers using the largest common integer size across different runtimes (u64)
- Specify large fees for chains which have a high gas cost

This allows us to use a common format across all chains, while being able to specify a large range of fees.

Not sure whether this is the right call, and it would be better to have differently-shaped payloads. That said, I'm not sure there's much downside to doing it this way either, apart from a small bit of added complexity.

Reply:
ah got it. that makes sense.